chrome-app/links.js

```diff
@@ -26,7 +26,7 @@ const LINKS = [
   [
     {
       title: "Review",
-      href: "https://github.com/pulls/review-requested",
+      href: gitHubQuery("https://github.com/pulls", [ ...pr, "review-requested:USERNAME", "-review:approved" ]),
```

Comment:
I believe this is implicit? I have approved PRs before and they have disappeared from my list.

Or maybe that is only if you approve it yourself...?

Reply:
In fact even just my responding to this PR with this review comment set that I did removes it from my "review requested" list.

Reply:
That hasn't been my experience. I totally see "approved" PRs in that list. could it be that you have `unread` enabled?

Reply:
I do not. But sure, I'll happily merge this